how-to-guides/data-versioning/notebooks/Organize_and_share_dataset_versions.ipynb

```diff
@@ -271,34 +234,18 @@
     "tags": []
    },
    "source": [
-    "## Step 3: See dataset versions in the Neptune UI and share them with the team\n",
+    "## See dataset versions in the Neptune UI and share them with the team\n",
     "\n",
     "You can get a list of all datasets used in a project by running ``project.get_structure()`` function."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
```

Comment:
maybe this structure can be copy&pasted as markdown cell -> it really helps understand this structure.